Sources/OVP/KalturaOVPPlayer.swift

```diff
@@ -269,10 +272,12 @@ extension KalturaOVPPlayer: EntryLoader {
             return
         }
         
+        ovpMediaOptions = mediaOptions
+        
         if let newKS = mediaOptions.ks, !newKS.isEmpty {
             updatePlayerOptionsKS(newKS)
         }
```

Comment:
Same here, now that the ovpMediaOptions is set, the didset performs the update ks, therefore it can be removed.

```suggestion
        ovpMediaOptions = mediaOptions
        
```